src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -303,8 +303,10 @@
     // Allows one to turn off cursors in compaction.
     CURSORS_ENABLED("cassandra.allow_cursor_compaction", "true"),
 
-    SYNC_LAG_FACTOR("cassandra.commitlog_sync_block_lag_factor", "1.5");
+    SYNC_LAG_FACTOR("cassandra.commitlog_sync_block_lag_factor", "1.5"),
 
+    CDC_STREAMING_ENABLED("cassandra.cdc.enable_streaming", "true");
```

Comment:
I didn't see this property set in cndb, but should point out that like other properties, I've changed the prefix from "dse" to "cassandra" here.

Reply:
I think this change is yes. Yes, it's not set in CNDB because we don't use the streaming code in CNDB. The idea is that this could be a guard in the future to prevent accidentally calling the code path in CNDB with CDC enabled. But right now it's unused :)